static/js/apps/eval_retrieval_generation/util.ts

```diff
@@ -206,13 +201,21 @@ function getEvalType(
 
 // Promise to get all the information about a google spreadsheet doc
 export function getDocInfo(doc: GoogleSpreadsheet): Promise<DocInfo> {
-  return getHeader(doc).then((allHeader) => {
-    return Promise.all([
-      getQueries(doc, allHeader),
-      getCalls(doc, allHeader),
-      getEvalType(doc, allHeader),
-    ]).then(([allQuery, allCall, evalType]) => {
+  return getHeader(doc)
+    .then((allHeader) => {
+      return Promise.all([
+        Promise.resolve(allHeader),
+        getEvalType(doc, allHeader),
```

Comment:
curious what the point of doing a separate promise.all here is when we getQueries and getCalls don't need to wait for evalType?

Reply:
ah, this is a holdover from when I was passing evalType into getCalls! revised.